### ISSUE TITLE
LNDhub implementation

### ISSUE THREAD

**Issue by ghost** [raspiblitz/raspiblitz#2434]
Really like the LNDhub implementations on myNode and Umbrel. Having an open source bank is dope and brings lots of scaling benefits to family uncle-Jim setups, with the clean look of Bluewallet's interface🙌. 

After this is complete i'll be migrating to the Raspiblitz platform as my main node of choice 🤝

**Comment by nyxnor** [comment]
If someone wants to work on this, there is this unfinished code I made to implement lndhub without docker https://github.com/rootzoll/raspiblitz/issues/230#issuecomment-847757874

**Comment by openoms** [comment]
Try the existing LNbits extension, it might work already.

**Comment by xanoni** [comment]
> If someone wants to work on this, there is this unfinished code I made to implement lndhub without docker [#230 (comment)](https://github.com/rootzoll/raspiblitz/issues/230#issuecomment-847757874)

This might fix it: https://github.com/rootzoll/raspiblitz/issues/230#issuecomment-890838807

**Comment by xanoni** [comment]
BTW I'm currently working on this, documenting progress here because there's been more discussion: https://github.com/rootzoll/raspiblitz/issues/230

**Comment by openoms** [comment]
@godSaysHODL did you try to install LNBits in which there is a functional LNDHub extension already?

Can also be tested on https://lnbits.com (this is a custodial demo wallet on Opennode). The biggest difference is that your own node would not be available on a public domain, but I think BW and Zeus can connect over Tor already.

@xanoni you could try this too.

**Comment by xanoni** [comment]
Yeah I saw your comment already a few weeks ago, but I had already set up LndHub, so I never gave it a try. The LndHub script is almost done so I'll get that across the finish line. If after all this time there's still someone willing to pay a bounty, please donate it to RaspiBlitz or to one of the upstream projects ;-)

**Comment by ghost** [comment]
@xanoni 

Definitely still gonna pay the bounty. Just ping me here when it's reviewed and looks like it'll be merged and I'll donate to whoever you choose!

**Comment by ghost** [comment]
> @godSaysHODL did you try to install LNBits in which there is a functional LNDHub extension already?
> 
> Can also be tested on https://lnbits.com (this is a custodial demo wallet on Opennode). The biggest difference is that your own node would not be available on a public domain, but I think BW and Zeus can connect over Tor already.

I will check this out also. 👌

**Comment by krtk6160** [comment]
> Yeah I saw your comment already a few weeks ago, but I had already set up LndHub, so I never gave it a try. The LndHub script is almost done so I'll get that across the finish line. If after all this time there's still someone willing to pay a bounty, please donate it to RaspiBlitz or to one of the upstream projects ;-)

Do you still plan on finishing this? LNDHub support would be awesome :)

**Comment by xanoni** [comment]
Oh man, I keep remembering this ticket all the time and feel a lot of guilt and shame (really!) ... thank you for pinging me. I'm not making promises because I already did that too often here and failed to deliver afterwards ;-)

There's no specific problem, there's just an infinite amount of other stuff. I just haven't prioritized time to continue.

**Comment by prusnak** [comment]
There is a very promising reimplementation of LndHub in Go language here: https://github.com/getAlby/lndhub.go
I recommend to use this as a LndHub/BlueWallet server.
